analysis_options.yaml

```
@@ -86,4 +86,4 @@ linter:
     unreachable_from_main: false
     implicit_call_tearoffs: false
     dangling_library_doc_comments: false
-    unintended_html_in_doc_comment: false
+    unintended_html_in_doc_comment: false
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)